test/functional/cmdLineTests/cmdLineTest_J9tests/j9tests.xml

```diff
@@ -44,5 +48,16 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
         <output type="failure" caseSensitive="yes" regex="no">Exception:</output>
     </test>
 
+    <test id="test -XX:-ReadIPInfoForRAS -XX:+ReadIPInfoForRAS">
+        <command>$EXE$ $NOREADIPINFOFORRAS$ $READIPINFOFORRAS$ -verbose:init -version</command>
+        <output type="success" caseSensitive="yes" regex="no">$READIPINFOFORRAS_MESSAGE$</output>
+        <output type="fail" caseSensitive="yes" regex="no">$NOREADIPINFOFORRAS_MESSAGE$</output>
```

Comment:
`type="fail"` is not supported. `"failure"` was correct.
https://github.com/eclipse/openj9/blob/0bd089b36679fe0e3c5bfd63df0fcfc22c16c36d/test/functional/cmdline_options_tester/src/TestCondition.java#L31